pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -1424,6 +1424,12 @@ public static class Minion {
     public static final char TASK_LIST_SEPARATOR = ',';
     public static final String CONFIG_OF_ALLOW_DOWNLOAD_FROM_SERVER = "pinot.minion.task.allow.download.from.server";
     public static final String DEFAULT_ALLOW_DOWNLOAD_FROM_SERVER = "false";
+
+    /**
+     * The number of threads to use for downloading segments from the deepstore.
+     * This is a global setting that applies to all tasks of BaseMultipleSegmentsConversionExecutor class.
+     */
+    public static final String SEGMENT_DOWNLOAD_THREAD_POOL_SIZE = "pinot.minion.task.segmentDownloadThreadPoolSize";
```

Comment:
this can be moved to MinionConf itself where other such minion confs are defined ?